pkg/sync/v7/derive.go

```diff
@@ -70,59 +67,3 @@ func (derivedType) StatsdArgs(cs *api.OpenShiftManagedCluster) ([]interface{}, e
 		"-SourceRoleInstance", "OSA",
 	}, nil
 }
-
```

Comment:
Why we removing this over multiple files? same as above

Reply:
This got moved from sync->startup